### DISCUSSION TITLE
Is someone using it with Astronvim?

### DISCUSSION THREAD

**Discussion by xmalter** [vscode-neovim/vscode-neovim#1195] [Q&A] [answered]
Error is `Unable to init vscode-neovim: command 'type' already exists`

**Comment by sol88pay** [comment, marked as answer]
maybe you installed VSCode extension that register the type command
https://github.com/vscode-neovim/vscode-neovim#troubleshooting

**Comment by dgrahamlong** [comment]
@xmalter You ever figure out how to use both? Can't get my mapping to work

**Reply by xiyaowong** [reply]
The error you encounter is caused by the vscode extension. You can only use one extension that needs to register the "type" command at the same time.
